Doc/faq/general.rst

```diff
@@ -160,8 +155,9 @@ unreleased versions, built directly from the CPython development repository.  In
 practice, after a final minor release is made, the version is incremented to the
 next minor version, which becomes the "a0" version, e.g. "2.4a0".
 
-See also the documentation for :data:`sys.version`, :data:`sys.hexversion`, and
-:data:`sys.version_info`.
+See the Developer’s Guide for more information about the development cycle, and
+PEP 387 to learn more about Python’s backward compatibility policy.  See also
+the documentation for sys.version, sys.hexversion, and sys.version_info.
```

Comment:
Make sure not to lose the reStructuredText links :)

Reply:
Yeah, just realized that. It should be fixed now. Thanks for catching!